Jenkinsfile

```diff
@@ -1,12 +1,52 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+stage('Trust') {
+    enforceTrustedApproval('aerogear')
+}
+
+def prLabels = getPullRequestLabels {}
+def metricsApiTemplateUrl = "https://gist.githubusercontent.com/psturc/a2a7150b66d48369e5fa62ab72352330/raw/91acde6f764e9bc59a84e3e34fa694206eb0a877/aerogear-app-metrics.yml"
```

Comment:
Should this come from a gist and from your account? Is not possible to put it in this repo?

Reply:
+1 on placing it in the repo. I would like to review the contents of the gist. Also, the app metrics service is still potentially subject to config and startup changes which would break this gist.

Reply:
Template added